### DISCUSSION TITLE
New Changes to Vlang.io

### DISCUSSION THREAD

**Discussion by Wajinn** [vlang/v#14892] [General]
Like that there is a latest news, very helpful.  But Changelog appears to be a "vulnerable spot", where visitors might get or act confused about what's going on.  Might be good to also update it, at least maybe every 3 months (quarterly).

Visitors opening the changelog, might get the impression that Vlang has not been updated since January 2021 or hasn't released 0.2.4 yet.  Changelog is also a very obvious part of the vlang.io page for people to click on.  

Any opinions about this?

**Comment by JalonSolov** [comment]
Agreed.  Just needs time for someone to put something in place.

**Comment by medvednikov** [comment]
@Wajinn I'm updating the changelog right now.

**Reply by Wajinn** [reply]
Well, that was one hell of an update.  Very nice!
